packages/bun-otel/README.md

```
@@ -0,0 +1,232 @@
+  traceExporter: new OTLPTraceExporter({
+    url: 'http://localhost:4318/v1/traces',
+  }),
+});
+
+await sdk.start();
+```
+
+### Distributed Tracing
+
+Client:
+```typescript
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)